### DISCUSSION TITLE
Not wrapping input at end of window

### DISCUSSION THREAD

**Discussion by xpapad** [scandum/tintin#220] [General]
I'm using TinTin++ 2.02.41 on a Mac terminal. When I type in and reach the end of the window, my input does not wrap to the next line; instead the beginning of the line shifts left and a < character appears in front. This makes it extremely hard to read my input. It doesn't happen with 2.01.4 on Ubuntu.
How can I turn this feature off and have my input wrap at the end of the line?

**Comment by scandum** [comment]
As far as I know this has always been the default behavior.

The only way to change it is to use #split with 5 arguments and make the input region at least 2 rows.

**Reply by xpapad** [reply]
This is surprising, I've been using tintin for over 25 years and this is the first time I see this. I'm wondering if it's a shell-specific issue. What would be the split arguments to support this?

**Comment by scandum** [comment]
`#split 0 1 0 0 2` should do the trick.

Possibly things changed if you're not using #split? I'd hate to mess with that code as the old method was causing other issues.

**Reply by xpapad** [reply]
Thanks. I'm not using split normally, but this will work.
